easybuild/easyconfigs/v/VSCode/VSCode-1.85.1-cli.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'Tarball'
+
+name = 'VSCode'
+version = '1.85.1'
+versionsuffix = '-cli'
```

Comment:
```suggestion
versionsuffix = '-cli-alpine-x64'
```

Reply:
Maybe `-cli-x64` is better, the `alpine` bit is actually a detail, and it would lead to confusion if we mention that in the module name (since this binary actually works on RHEL8)